com.unity.cinemachine/Runtime/Helpers/CinemachineBrainEvents.cs

```diff
@@ -1,12 +1,11 @@
-using System;
 using UnityEngine;
-using UnityEngine.Events;
 
 namespace Unity.Cinemachine
 {
     /// <summary>
-    /// This component will generate brain-specific events.
-    /// Add it to a a GameObject that has a CinemachineBrain.
+    /// This component will generate mixer-specific events.
+    /// Add it to a a GameObject that has a CinemachineBrain or another ICinemachineMixer, 
+    /// such as a CinemachineCameraManager.
     /// </summary>
     [AddComponentMenu("Cinemachine/Helpers/Cinemachine Brain Events")]
     [SaveDuringPlay]
```

Comment:
Is BrainEvents a good name? Now it is more generic.

Reply:
It's not the best name, but since it will be used most often on the brain I thought I would leave the name, for discoverability

Reply:
Ok 👍